### DISCUSSION TITLE
wait_on_done

### DISCUSSION THREAD

**Discussion by NDR008** [trackmania-rl/tmrl#72] [Q&A]
I'm not sure, if I am using this right, but I assumed that when wait_on_done is true, the environment would not reset on done?
Perhaps I am miss-using it because my env-restarts :(

**Comment by yannbouteiller** [comment]
As implied by the name, it calls the wait() method of rtgym when the environment is done, then whatever you implemented as this wait() method happens.

**Reply by NDR008** [reply]
AH! Makes sense.
